projects/angular/src/forms/styles/_select.clarity.scss

```
@@ -120,84 +104,40 @@
   //Multiple Select
   select[multiple],
   select[size] {
-    $multiSassBgVar: forms-variables.$clr-forms-select-multiple-background-color;
-    $multiCssBgVar: clr-forms-select-multiple-background-color;
-    $multiSassBorderVar: forms-variables.$clr-forms-select-multiple-border-color;
-    $multiCssBorderVar: clr-forms-select-multiple-border-color;
     padding: 0;
-    @include mixins.css-var(background, $multiCssBgVar, $multiSassBgVar, variables.$clr-use-custom-properties);
     border: variables.$clr-global-borderwidth solid;
-    @include mixins.css-var(
-      border-color,
-      $multiCssBorderVar,
-      $multiSassBorderVar,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      border-radius,
-      clr-global-borderradius,
-      variables.$clr-global-borderradius,
-      variables.$clr-use-custom-properties
-    );
+    background: forms-variables.$clr-forms-select-multiple-background-color;
+    border-color: forms-variables.$clr-forms-select-multiple-border-color;
```

Try to combine border in one rule.

```suggestion
    border: tokens.$cds-alias-object-border-width-100 solid forms-variables.$clr-forms-select-multiple-border-color;
```